src/java.desktop/share/classes/java/awt/Window.java

```diff
@@ -3949,7 +3950,7 @@ public void paint(Graphics g) {
             try {
                 if (gg instanceof Graphics2D) {
                     gg.setColor(getBackground());
-                    ((Graphics2D)gg).setComposite(AlphaComposite.getInstance(AlphaComposite.SRC));
+                    ((Graphics2D) gg).setComposite(AlphaComposite.getInstance(AlphaComposite.SRC));
```

Comment:
Why was the import for SunHints added if the only edit to this file was adding a space?

Reply:
Nevermind I saw this wrong when looking through the commit history.

Reply:
Thanks for checking, though. I just pushed a change to roll back this diffs. https://github.com/openjdk/jdk/pull/12993/commits/f369ccde6d73537948d6f5007ae5d0a2b847d8ac